include/net/nrf_cloud_rest.h

```diff
@@ -0,0 +1,259 @@
+/*
```

Comment:
Why is this part of this PR? Not needed for AWS IoT.

Reply:
It's part of #4609 which this PR is based on. Same goes for all the REST files.